library/core/src/macros/panic.md

```diff
@@ -9,11 +9,14 @@ tests. `panic!` is closely tied with the `unwrap` method of both
 [`Option`][ounwrap] and [`Result`][runwrap] enums. Both implementations call
 `panic!` when they are set to [`None`] or [`Err`] variants.
 
-This macro is used to inject panic into a Rust thread, causing the thread to
-panic entirely. This macro panics with a string and uses the [`format!`] syntax
-for building the message.
-
-Each thread's panic can be reaped as the [`Box`]`<`[`Any`]`>` type,
+When using `panic!()` you can specify a string payload, that is built using
+the [`format!`] syntax. That payload is used when injecting the panic into
+the calling Rust thread, causing the thread to panic entirely.
+
+The default `std` panic handling strategy is to print the message payload to
```

Comment:
The panic *handling* stragety of `std` is to unwind or abort. Printing to stderr is done as part of the panic *hook*, which executes before unwinding/aborting happens. Even if you handle a panic yourself with `catch_unwind` or `thread.join()`, the hook was already executed and already printed the panic to stderr.

Reply:
Okay, I've chosen the wrong terminology. Should be fixed.